src/work-allocation/components/task-assignment/task-assignment.component.ts

```diff
@@ -75,25 +78,67 @@ export class TaskAssignmentComponent implements OnInit {
     return this.pCaseworkers;
   }
 
+  /**
+   * Gets the private logged in user id
+   */
+  public get userId(): string {
+    return this.pUserId;
+  }
+
+  /**
+   * Sets the private logged in user id
+   */
+  public set userId(value: string) {
+    if (this.pUserId !== value) {
+      this.pUserId = value;
+    }
+  }
+
+  /**
+   * Gets the private logged in user id
+   */
+  public get assignedCaseworker(): Caseworker {
+    return this.pAssignedCaseworker;
+  }
+
+  /**
+   * Sets the private logged in user id
+   */
+  public set assignedCaseworker(value: Caseworker) {
```

Comment:
Same applies to this - this will allow the assigned caseworker to be overridden and circumvent `setupAssignedCaseworker()`. In this instance, it looks like that might be not be a terrible idea - i.e., you can assign to someone else - but, again, this _looks_ like it's just for the benefit of unit tests...?

Reply:
Will remove setter (will leave comment stating could be useful)